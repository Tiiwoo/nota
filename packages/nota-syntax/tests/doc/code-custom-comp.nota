%let f = ({x}) => @foo[attr: @bar{baz}]{#x}
@f[x: "y"]