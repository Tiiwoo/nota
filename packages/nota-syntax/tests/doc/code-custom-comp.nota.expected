[
  ...(() => {
    let f = ({ x }) =>
      el(
        foo,
        {
          block: true,
          attr: el(
            bar,
            {
              block: true,
            },
            "baz"
          ),
        },
        x
      );

    return [
      null,
      el(f, {
        block: true,
        x: "y",
      }),
    ];
  })(),
];